includes/class-wc-ajax.php

```diff
@@ -2642,7 +2737,7 @@ public static function shipping_zone_methods_save_settings() {
 		$instance_id     = absint( $_POST['instance_id'] );
 		$zone            = WC_Shipping_Zones::get_zone_by( 'instance_id', $instance_id );
 		$shipping_method = WC_Shipping_Zones::get_shipping_method( $instance_id );
-		$shipping_method->set_post_data( $_POST['data'] );
+		$shipping_method->set_post_data( wp_unslash( $_POST['data'] ) ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
Is this getting sanitized later or not needed here?

Reply:
It's handled in `process_admin_options`. It expects raw post data - we're just stripping slashes here.